### ISSUE TITLE
unordered_writes in local-config causes error

### ISSUE THREAD

**Issue by TomRitserveldt** [grafana/loki#4305]
**Describe the bug**
this PR: https://github.com/grafana/loki/pull/4149 introduced the `unordered_writes: true` setting in the limits_config.
however, running loki 2.3.0, this throws an error on startup:
```
failed parsing config: loki-local-config.yaml: yaml: unmarshal errors:
  line 50: field unordered_writes not found in type validation.plain
```

**To Reproduce**
1. download and extract loki 2.3.0 on debian
2. download the latest loki-local-config.yaml from this repo
3. start loki with said config: `./loki-linux-amd64 -config.file=loki-local-config.yaml`

**Expected behavior**
loki starts.

**Environment:**
 - Infrastructure: debian buster VM
 - Deployment tool: manual install.

**Screenshots, Promtail config, or terminal output**
```
# ./loki-linux-amd64 -config.file=loki-local-config.yaml
failed parsing config: loki-local-config.yaml: yaml: unmarshal errors:
  line 50: field unordered_writes not found in type validation.plain
```

**Comment by owen-d** [comment]
Thanks for your interest @TomRitserveldt! Unfortunately, that configuration is not available in 2.3, but will be in 2.4 when it's released :)

**Comment by klw123** [comment]
so , how to solve this problem?? remove the code in line 50 of field unordered_writes not found in type validation.plain?
